<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calorie Counter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Calories Burned: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="caloryCountLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0)</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Time (hours): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="timeSpinBox">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Slope: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="slopeComboBox">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>-5% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-4% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-3% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-2% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-1% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Level</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+1% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+2% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+3% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+4% grade</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+5% grade</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Weight(kg): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="weightSpinBox">
         <property name="value">
          <double>70.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed(km/hr);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="speedSpinBox">
         <property name="value">
          <double>6.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
